cmake/kokkos_arch.cmake

```diff
@@ -147,6 +147,7 @@ if(KOKKOS_ENABLE_COMPILER_WARNINGS)
       "-Wsign-compare"
       "-Wtype-limits"
       "-Wuninitialized"
+      -Wsuggest-override
```

Comment:
```suggestion
      "-Wsuggest-override"
```

Reply:
The quotes are superfluous in this case. It would have been nice if your `CMake` tidy had told you that :angel:

Reply:
I know they are but this is about consistency

Reply:
OK.